content.js

```diff
@@ -179,11 +179,36 @@ document.onkeydown = function(e){
 }
 
 function videoIDChange(id) {
+
   //not a url change
-  if (sponsorVideoID == id){
-    return;
-  }
+  if (sponsorVideoID == id) return;
+  
+  chrome.storage.sync.set({videoid: id});
+
+  //warn them if they had unsubmitted times
+  if (previousVideoID != null) {
```

Comment:
Content script can do notifications?

Reply:
Not sure may need to create API I think the code works better per tab

Reply:
Added alertPrevious message to background